openshift/templates/api/api-deploy.yaml

```diff
@@ -120,6 +120,7 @@ objects:
       oidc-rp-client-secret: ${OIDC_RP_CLIENT_SECRET}
       oidc-rp-host: ${OIDC_RP_HOST}
       encryption-key: ${DATA_SECURITY_KEY}
+      efiling-enabled: ${EFILING_ENABLED}
```

Comment:
Since this setting does not contain any sensitive information there is no need to store it as a secret.